<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCC (working title)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="OpenGLScene" name="sceneWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>731</width>
      <height>421</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="63">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QWidget" name="modeParent" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <zorder>sceneWidget</zorder>
   <zorder>modeParent</zorder>
   <zorder>mdiArea</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLScene</class>
   <extends>QOpenGLWidget</extends>
   <header>openglscene.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
